app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -77,42 +77,29 @@ class ExoPlayerProfile(
 						CodecTypes.MPEG2VIDEO
 					).joinToString(",")
 
-					audioCodec = if (Utils.downMixAudio()) {
-						arrayOf(
-							CodecTypes.AAC,
-							CodecTypes.MP3,
-							CodecTypes.MP2
-						).joinToString(",")
-					} else {
-						listOfNotNull(
-							CodecTypes.AAC,
-							CodecTypes.AC3,
-							CodecTypes.EAC3,
-							CodecTypes.AAC_LATM,
-							CodecTypes.MP3,
-							CodecTypes.MP2,
-							CodecTypes.DCA,
-							CodecTypes.DTS,
-							CodecTypes.OPUS,
-						).joinToString(",")
-					}
+					audioCodec =
+						if (Utils.downMixAudio())
+							getDownmixSupportedAudioCodecs().joinToString(",")
+						else
+							getAllSupportedAudioCodecs().joinToString(",")
```

Comment:
```suggestion
					audioCodec = if (Utils.downMixAudio()) getDownmixSupportedAudioCodecs().joinToString(",")
						else getAllSupportedAudioCodecs().joinToString(",")
```

A when statement might look better here

Reply:
That's much more readable, I'll try to get used to using when more often :+1: